dev/com.ibm.ws.jaxrs.2.x.webcontainer_fat/fat/src/com/ibm/ws/jaxrs2x/fat/helloworld/HelloWorldTest.java

```diff
@@ -0,0 +1,176 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package com.ibm.ws.jaxrs2x.fat.helloworld;
+
+/*
+ * IBM Confidential
+ *
+ * OCO Source Materials
+ *
+ * Copyright IBM Corp. 2011
+ *
+ * The source code for this program is not published or other-
+ * wise divested of its trade secrets, irrespective of what has
+ * been deposited with the U.S. Copyright Office.
+ */
```

Comment:
I think this is what the copyright tool is complaining about.  Can you delete these comments?  The copyright header at the top looks right.

Reply:
Done